### PR TITLE
[Improvement][E2E] Add dolphinscheduler-e2e into root pom

### LINKED ISSUE #15224: [Improvement][E2E] Add dolphinscheduler-e2e into root pom
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Right now, if we want to add e2e case or run the e2e case locally, it's not convenient, can we add dolphinscheduler-e2e into root pom?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pegasas**
## Purpose of the pull request

resolves https://github.com/apache/dolphinscheduler/issues/15224

## Brief change log

Add dolphinscheduler-e2e into root pom.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
         <module>dolphinscheduler-aop</module>
         <module>dolphinscheduler-dao</module>
         <module>dolphinscheduler-dist</module>
+        <module>dolphinscheduler-e2e</module>
```
Comment (SbloodyS) [inline pom.xml]: We should prevent the e2e module from participating in normal compilation.

**Review by SbloodyS — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (pegasas) [reply pom.xml]: So it seems like we can close this issue? 
due to having other concern on this.

Comment (ruanwenjun) [reply pom.xml]: Can we use profile to prevent execute e2e case in normal compilation?

Comment (pegasas) [reply pom.xml]: Good suggestion.
Already add module with profile.

**Comment by zhongjiajie** [comment]
keep the old org makes more sense in our project because we do not want to compile both e2e and api test to tarball, but we still thanks for your contribution

**Pull request closed without merging**